src/interfaces/python/swig_typemaps.i

```diff
@@ -1371,9 +1373,9 @@ def _internal_factory_wrapper(object_name, new_name, docstring=None):
     via .put
     """
     _obj = getattr(sys.modules[__name__], object_name)
-    def _internal_factory(name, **kwargs):
+    def _internal_factory(name, *args, **kwargs):
```

Comment:
It might be worth mentioning in the docstring that *args are passed to the object at instantiation?
It's a good idea to handle constructors with multiple arguments!

Reply:
agreed!

Reply:
Not needed abymore